TestCases/fea_fsi/WallChannel_2d/configFSI.cfg

```diff
@@ -22,6 +22,8 @@ RESTART_ITER = 2
 
 OUTER_ITER = 5
 
+MATCHING_MESH = YES
```

Comment:
This might be the reason why tests are not passing.

Reply:
Hi @pcarruscag 
ok, then, why is it necessary to specify MATCHING_MESH in the config?

Also, given that for the test cases the meshes are matching, I don't understand why the results should change (independently of the mesh being coarser or finer). It should not matter to do either
- Compute stresses -> integrate forces -> transfer data -> apply loads
- Compute stresses -> transfer data -> integrate forces -> apply loads

as in both sides the areas are exactly the same.

Reply:
It does not matter at convergence, complementing what I wrote on Oct 12:
On the fluid side the area vector is available at the node.
On the structural side I am first integrating nodal stresses which results in a force at face centers, then those need to be put back on the nodes. Which means, that for a surface mesh of quads, each nodal force depends on 9 nodal stresses instead of 1.

Reply:
For clarity, I mean grid convergence.

Reply:
I understand what you mean. However, the tests were originally passing with these changes, and the same mesh and settings, see, e.g., here:
https://travis-ci.org/su2code/SU2/builds/460157253
and then they failed in the next build:
https://travis-ci.org/su2code/SU2/builds/461388095
which only included the commit
https://github.com/su2code/SU2/commit/b4638d639c3a93a95b4b1d8647a40c6c36b64a39
into the develop after the new Driver structure was merged in.

Provided that the new driver is recovering the same behaviour as before (and that's the case, because the tests passed), I still don't understand what's the fundamental change that's made the regression tests fail now and wasn't making them fail before. There should be possible to make them pass without any modification of the test values.

Reply:
They were passing before the changes to the driver, it started failing here:
https://travis-ci.org/su2code/SU2/builds/460835123
Because SU2 is using nearest neighbour consistent interpolation and the option matching mesh is not taking effect on the multiple config cases.
You can see that before b4638d6 two cases were failing (one adjoint one direct) and after I put the matching mesh option in the configs the adjoint passed (single config) while the other did not.

Reply:
I see. Doesn't that mean though that setting
`CONSERVATIVE_INTERPOLATION = YES`
should do the trick? It should probably be the default anyway.

Reply:
I totally agree with that, it's also on my list to remove some options that might be unnecessary and/or duplicate, this is a good example. Could you remove `MATCHING_MESH=YES` in the FSI regression tests by making CONSERVATIVE_INTERPOLATION=YES as the default? They should pass like that. Maybe adding a consistent, non-matching simple FSI test case (direct and adjoint) would be useful for completeness. I'll also have a look as soon as I have a minute to other test cases to remove that keyword, and maybe some other confusing ones in a separate PR.

Reply:
@rsanfer I made the changes to the settings and if you agree I changed the airfoil RBF testcase to consistent interpolation.
I will open a PR in the TestCase repo to update the initial conditions for the case.

Reply:
I think it's a good idea to use the airfoil RBF as testcase for consistent interpolation.
However, the original test case keeps failing, and with conservative interpolation enabled and a matching mesh, that should not be the case. Could you check it out?

Reply:
I did not put the option in all the configs, still getting to grips with the multi-config process. In any case the change to the default value should do the trick.
I merged the TestCases PR so the RBF case should pass too.
Please advise on the enumerator subject as this should be good to go.